client/valor/client.py

```diff
@@ -404,7 +403,7 @@ def get_labels(
         """
         Gets all labels using an optional filter.
 
-        `GET` endpoint.
```

Comment:
remind me: why is it necessary to remove filter query params, which in turns makes this `GET` request into a `POST` request?

Reply:
Yes. B/c the new filters can sometimes be larger than is allowed in query params.